src/datepicker/DayPicker.vue

```diff
@@ -132,8 +136,12 @@ export default defineComponent({
     const isEnabled = (
       target: Date,
       lower: Date | undefined,
-      upper: Date | undefined
+      upper: Date | undefined,
+      disabledDates: { dates: Date[] | undefined } | undefined
     ): boolean => {
+      if (disabledDates
+          && disabledDates.dates
+          && disabledDates.dates.some(date => isSameDay(target, date))) return false
```

Comment:
Since we're transpiling the code anyway, we could just use `?.` accessor to skip extensive checks